metadata.yaml

```diff
@@ -2,7 +2,7 @@
 # See LICENSE file for licensing details.
 name: mlflow-server
 description: |
-  MLflow
+  mlflow is a Kubernetes charm bundle for MLflow. This bundle automates the deployment and operation of MLflow on any Kubernetes cloud.
```

Comment:
not sure about Cloud

K8s cluster maybe ?

Reply:
Agree with @beliaev-maksim we can say **on any Kubernetes cluster backed by any public cloud**. Just wanted to say we have never tested it yet on **any public cloud**, we are just starting :D So far we are just testing on microk8s.

Reply:
Makes sense. I kept it safe. Let me know if that works.